test/jasmine/tests/mock_test.js

```diff
@@ -959,6 +959,9 @@ var list = [
     'tick-percent',
     'tickformat',
     'tickformatstops',
+    'ticklabelposition-0',
+    'ticklabelposition-1',
+    'ticklabelposition-2',
```

Comment:
Oh I forgot about this step in #5287 - would you mind adding `axes_chain_scaleanchor_matches` and `axes_chain_scaleanchor_matches2` for me?

Reply:
Done in https://github.com/plotly/plotly.js/pull/5275/commits/91ea2f616a092189d78c2334b5602b0327507dab.